content/es/governance.md

```diff
@@ -0,0 +1,137 @@
+
+Una guía para los Colaboradores es mantenida en
+[COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
+
+## CT Membresía
+
+Los puestos del CT no son de duración limitada. No hay un tamaño fijo del
+CT. Sin embargo, el tamaño esperado es de 6 a 12 para asegurar una
+adecuada cobertura de las áreas importantes de experiencia en
+equilibrio con la capacidad de tomar decisiones eficientemente.
+
+No hay un conjunto específico de requisitos o calificaciones para ser miembro
+del CT más allá de estas reglas.
+
+El CT puede agregar miembros adicionales al CT por una moción estándar
+del TC.
+
+Un miembro de CT puede ser removido de la CT por renuncia voluntaria, o por una
+moción estándar del CT.
+
+Cambios a la membresía del CT deberían ser publicados en la agenda y
+pueden ser sugeridos como cualquier otro punto de la agenda (ver "CT Reuniones" abajo).
+
+No más de un tercio de los miembros del CT puede estar afiliados con el
+mismo empleador. Si la remoción o renuncia de un miembro del CT o un
+cambio de empleo de un miembro del CT crea una situación en la que más de
+un tercio de las membresías del CT comparten un empleador, entonces la
+situación debe remediarse inmediatamente por la renuncia o remoción de uno
+o más miembros del CT afiliados con el empleador(es) sobre-representado.
+
+## TC Reuniones
+
+El CT se reúne semanalmente en Google Hangout On Air. La reunión está
+dirigido por un moderador designado y aprobado por el CT. Cada reunión
+debe ser publicada en YouTube.
+
+Los asuntos que se agregan a la agenda del CT son considerados discutibles
+o son modificaciones de gobernabilidad, política de contribución,
+membresía del CT o del proceso de liberación.
+
+La intención de la agenda no es la de aprobar o revisar todos los parches,
+eso debe ocurrir continuamente en GitHub y ser manejado por el amplio grupo
+de colaboradores.
+
+Cualquier miembro de la comunidad o contribuyente puede pedir que algo se
+agrege a la agenda de la próxima reunión ingresando un asunto en GitHub.
+Cualquier Colaborador, miembro del TC o el moderador puede agregar el punto
+ a la agenda agregando la etiqueta ***tc-agenda*** al asunto.
+
+Antes de cada reunión del CT el moderador compartirá la Agenda a los
+miembros del CT. Los miembros del CT pueden agregar cualquier punto que
+deseen a la agenda al inicio de cada reunión. El moderador y el CT no
+pueden vetar o eliminar elementos.
+
+El CT podrá invitar a las personas o representantes de determinados proyectos
+a participar sin derecho de voto. Estos invitados actualmente son:
+
+* Un representante de [build](https://github.com/node-forward/build)
+  elegido por ese proyecto.
+
+El moderador es responsable de resumir la discusión de cada punto de
+la agenda y enviarlo como una solicitud de extracción después de la reunión.
+
+## Proceso de Búsqueda de Consenso
+
+El CT sigue un
+[Buscando el Consenso](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+modelo de toma de decisiones.
```

Comment:
* What do you think about the following translation?
`El CT sigue un modelo de toma de decisiones basado en la [Búsqueda del Consenso](...)`